examples/plotting/plot_3_lpi.py

```diff
@@ -0,0 +1,159 @@
+# local parameter importance.
+# The error bars represents the standard deviation of the LPI based on 10 runs. Remember
+# that the LPI requires the training of a regression model. The initial state of this model can
+# greatly influence the results. This is why the computation of the LPI is done multiple times
+# (10 times by default) using different random seeds. Here is an example
+# setting the number of points for the grids, the number of runs and the initial random seed
+# for the regression model.
+
+experiment.plot.lpi(n_points=10, n_runs=5, model_kwargs=dict(random_state=1))
+
+#%%
+# We can see that the learning rate had a larger impact than the dropout in achieving the best
+# objective.
+# A search space of only 2 dimensions is easy to analyse visualy however so the LPI
+# has little additional value in this example. We need to use an example with a larger search space
+# to better show to utility of the LPI. We will load results from tutorial
+# :ref:`sphx_glr_auto_tutorials_code_2_hyperband_checkpoint.py` for this.
+
+# Load the data for the specified experiment
+experiment = get_experiment("hyperband-cifar10", storage=storage)
+experiment.plot.lpi()
+
+#%%
+# There is a large difference here between the most important hyperparameter (learning rate) and the
+# least important one (gamma).
+#
+# One caveat of LPI is that the variance for each hyperparameters depends on the search space.
+# If the prior for one hyperparameter is narrow and fits the region of best values for this
+# hyperparameter, then the variance will be low and this hyperparameter will be considered
+# non-important. It may be important, but it is not important to optimize it within this narrow
+# search space. Another related issue, is that if one hyperparameter have a dramatic effect, it will
+# lead to a variance so large that the other hyperparameters will seem unrelevant in comparison.
+# This is what we observe here with the learning rate. If we branche from the experiment and define
+# a narrowed prior for the learning rate, we will see that it becomes an unimportant
+# hyperparameter.
+# See documentation on :ref:`EVC system` for more information on branching, or
+# :py:func:`orion.client.build_experiment` for informations on ``branching`` arguments.
+# Original learning rate prior was ``loguniform(1e-5, 0.1)``. We will narrow it to
+# ``loguniform(1e-3, 0.1)``.
+
+from orion.client import build_experiment
+
+# Branch from "hyperband-cifar10" with a narrower search space.
+experiment = build_experiment(
+    "narrow-hyperband-cifar10",
+    branching={"branch_from": "hyperband-cifar10"},
+    space={
+        "epochs": "fidelity(1, 120, base=4)",
+        "learning_rate": "loguniform(1e-3, 0.1)",
+        "momentum": "uniform(0, 0.9)",
+        "weight_decay": "loguniform(1e-10, 1e-2)",
+        "gamma": "loguniform(0.97, 1)",
+    },
+    storage=storage,
+)
+
+experiment.plot.lpi()
```

Comment:
No data appears to be displayed in the plot.

Reply:
Seems ok on my side in current version. I will leave this one as unresolved and double-check if the plot is fine when I push my modifications.